DOCS.md

```diff
@@ -1,19 +1,52 @@
-Use the PyPI plugin to deploy a Python package to a PyPI server.
+Use the PyPi plugin to deploy a Python package to a PyPI server. You can
+override the default configuration with the following parameters:
 
-* `repository` - The repository name (optional)
-* `username` - The username to login with (optional)
-* `password` - A password to login with (optional)
-* `distributions` - A list of distribution types to deploy (optional)
+* `username` - Username to login
+* `password` - Password to login
 
-The following is an example configuration for your .drone.yml:
+## Example
+
+The following is a sample configuration in your .drone.yml file:
+
+```yaml
+publish:
+  pypi:
+    username: guido
+    password: secret
+```
+
+### Source dist
```

Comment:
`sdist` is the default, so in this case it's equivalent to leaving the `distributions` parameter empty. A `bdist_wheel` example might be more meaningful here.